lib/truncate.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (messageText, maxLength, delimiter) => {
+	// default delimiter is newline
+	delimiter = delimiter || '\n'
```

Comment:
As the delimiter is always `\n` I think the delimiter parameter is unnecessary. I thought about the case where the delimiter is `\r\n` but it should work exactly as intended, as it will still split (and join) on `\n`.